lib/envs.js

```diff
@@ -138,6 +139,20 @@ function loadEnvs(path) {
   return env;
 }
 
+function loadEnvs(path) {
+  if (path.indexOf(',') === -1) {
```

Comment:
Is this needed? I think `path.split(',')` below will handle this case as a single item Array and "just work", won't it?